pathfinder/model/token_network.py

```diff
@@ -106,6 +106,19 @@ def handle_channel_closed_event(self, channel_identifier: ChannelIdentifier):
                 ),
             )
 
+    @staticmethod
+    def edge_weight(
+        visited: Dict[ChannelIdentifier, float],
```

Comment:
Is the `float` annotation right here? From what I see only `DIVERSITY_PEN_DEFAULT` is added to this, so it might be an int?

Reply:
Right now they are ints, but I don't think we need to restrict ourselves to ints. We certainly don't want to use `Union[int, float]` in such a case.